.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "standard",
   "rules": {
-    "no-labels": 0
+    "no-labels": 0,
+    "padded-blocks": 0,
```

Comment:
@rjrodger @geek I'm going to need this rule if we want to get better at commenting otherwise we are forced to bunch everything up.